crates/aptos/src/common/init.rs

```diff
@@ -101,7 +101,7 @@ impl CliCommand<()> for InitTool {
 
         // Select profile we're using
         let mut profile_config = if let Some(profile_config) = config.remove_profile(profile_name) {
-            prompt_yes_with_override(&format!("Aptos already initialized for profile {}, do you want to overwrite the existing config?", profile_name), self.prompt_options)?;
+            prompt_yes_with_override(&format!("Movement already initialized for profile {}, do you want to overwrite the existing config?", profile_name), self.prompt_options)?;
```

Comment:
You have updated the directory output to be `.movement`, right?

Reply:
Yes, I just added [a commit](https://github.com/movementlabsxyz/aptos-core/pull/49/commits/87b7ef250c181a8937c472f08af5c962cd7fa2eb) to update it.

Reply:
Okay, let me test this e2e and then hopefully I will be able to approve.